### PR TITLE
Hide non-data rows from bulk edit form

### PR DESCRIPTION
**Pull request by magicznyleszek**
## Description

The `hidden`, `note` and `calculate` questions don't carry submission data, so we hide them from the Bulk Edit Submissions form (and Single Submission Modal).

### CODE REVIEW

File: jsapp/js/components/modalForms/bulkEditSubmissionsForm.es6 [hunk 1]
```diff
@@ -173,10 +176,23 @@ class BulkEditSubmissionsForm extends React.Component {
...
    */
   getDisplayData() {
-    const questions = getFlatQuestionsList(this.props.asset.content.survey);
+    let questions = getFlatQuestionsList(this.props.asset.content.survey);
     const flatPaths = getSurveyFlatPaths(this.props.asset.content.survey);
 
-    questions.forEach((question) => {
+    questions = questions.filter((question) => {
+      // we don't want to include special calculate row used to store form version
+      if (question.type === QUESTION_TYPES.calculate.id && question.name === FORM_VERSION_NAME) {
```
Comment (duvld) [inline jsapp/js/components/modalForms/bulkEditSubmissionsForm.es6:184]: There is a small edge case that bypasses this check: If a form has multiple versions like [this one](https://github.com/kobotoolbox/kpi/files/6131190/aEYQ8CkLw3QKmynaXUAw8U.xlsx) for example

**Comment by duvld** [comment]
Should we put this on master as well? :D

File: jsapp/js/components/modalForms/bulkEditSubmissionsForm.es6 (hunk 1 above)
Comment (magicznyleszek) [reply jsapp/js/components/modalForms/bulkEditSubmissionsForm.es6]: This is a great comment. It actually makes more sense to just hide all `calculate` rows here (and in the single submission modal)!

**Comment by magicznyleszek** [comment]
> @magicznyleszek Should we put this on master as well? :D

Nah, Bulk Edit is on `beta` only, and the changes I did for the Single Submission Modal are really tiny

**Pull request merged**